libsnark/zk_proof_systems/plonk/circuit.hpp

```diff
@@ -0,0 +1,74 @@
+/** @file
+ *****************************************************************************
+ * @author     This file is part of libff, developed by Clearmatics Ltd
+ *             (originally developed by SCIPR Lab) and contributors
+ *             (see AUTHORS).
+ * @copyright  MIT license (see LICENSE file)
+ *****************************************************************************/
+
+#ifndef LIBSNARK_ZK_PROOF_SYSTEMS_PLONK_CIRCUIT_HPP_
+#define LIBSNARK_ZK_PROOF_SYSTEMS_PLONK_CIRCUIT_HPP_
+
+#include "libsnark/zk_proof_systems/plonk/plonk.hpp"
+
+/// Declaration of Common Preprocessed Input interfaces for ppzkSNARK
```

Comment:
Maybe just make it clear that this is for a specific circuit?
```suggestion
/// Declaration of Common Preprocessed Input data structures for a specific arithmetic circuit.
```

Reply:
Change applied.